remote_branch_checker/remote_branch_checker.sh

```diff
@@ -146,6 +146,11 @@ basecommit=$(${gitcmd} rev-parse --verify ${baseref})
 # (NOTE: checkout -B means create if branch doesn't exist or reset if it does.)
 ${gitcmd} checkout -q -B ${integrateto}_precheck $baseref
 
+# Do some cleanup onto the passed details
+
+# Trim whitespace in branch/remote
```

Comment:
Yeah. I think, my initial idea was to trim, but it was easier just to get rid of all spaces.

* As it needs to be a url the remote can't have spaces
* neither can branch: https://www.kernel.org/pub/software/scm/git/docs/git-check-ref-format.html